server/scripts/amqp.js

```diff
@@ -49,6 +46,36 @@ amqp.connect(config.rabbitmq.host)
               // send to angular clients
               wss.broadcast(JSON.stringify(insertedRow));
             });
+
+          // Process Lap data
+          // Want to capture when button is released
+          // TODO: Swap VolumeDown with lap button when it's ready
+          if (!jsonObj.DriverControls.VolumeDown
+            && volumeDown) {
+            db.between(lastLapTimestamp, jsonObj.TimeStamp)
+               .then((allPackets) => {
+                 let averagePackCurrent = rc.getAveragePackCurrent(allPackets);
+                 let amphours = jsonObj.Battery.PackAmphours;
+                 let batterysecondsremaining = rc.getSecondsRemainingUntilChargedOrDepleted(averagePackCurrent, amphours);
```

Comment:
Average pack current uses camelcase while amphours and batterysecondsremaining uses lowercase, was this intended?

Reply:
Nope, good catch!

Reply:
lgtm